CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
```

Comment:
The new 'Salary' property is added in between the [ForeignKey] attribute and the foreign key navigation property. This breaks the association mapping for Entity Framework. It must remain directly above the 'CompanyId' property.

```suggestion
public decimal Salary { get; set; }
```